tests/test_audio_chroma.py

```diff
@@ -10,12 +10,12 @@
 import numpy as np
 import unittest
 from . import AUDIO_PATH, ACTIVATIONS_PATH
-from madmom.audio.chroma import DeepChromaProcessor
+from madmom.audio.chroma import DeepChromaProcessor, CLPChroma
 from madmom.features import Activations
 
-
 sample_file = "%s/sample.wav" % AUDIO_PATH
-sample_act = Activations("%s/sample.deep_chroma.npz" % ACTIVATIONS_PATH)
+sample_act_deep_chroma = Activations("%s/sample.deep_chroma.npz" %
+                                     ACTIVATIONS_PATH)
```

Comment:
Since you have to rebase this code anyways, please make sure you use `os.path.join` or the `pj()` function instead of `%s/` to be compatible with windows (see other tests).